### ISSUE TITLE
Use bigger font size (at least when a caps lock / num lock warning is shown)

### ISSUE THREAD

**Issue by CodingSpiderFox** [elementary/greeter#345]
The capslock and numlock notifications are something I missed for a long time. Thanks for adding it!

But the font is very small. I suggest making all fonts bigger by +8pt on the greeter.

Maybe this can be made configureable with gsettings?

**Comment by J0Ar** [comment]
Yes, at least it should respect the setting of the text-scaling-factor of the desktop… 
_org.gnome.desktop.interface text-scaling-factor XXX_

**Comment by cassidyjames** [comment]
@CodingSpiderFox what resolution and size of display are you using? Are you using text scaling in your user session? I believe those labels are the same size as the rest of the standard UI font, so they shouldn't be any smaller than everything else.

**Comment by CodingSpiderFox** [comment]
Screen resolution = 1920x1080
Where can I check for the text scaling settings?

**Comment by cassidyjames** [comment]
@CodingSpiderFox _System Settings_ → _Desktop_ → _Appearance_

**Comment by CodingSpiderFox** [comment]
Text size is set to default there

**Comment by cassidyjames** [comment]
Hm. Do you find other text in the OS too small? I do see we're using a `<small>` tag here, maybe we can drop that:

https://github.com/elementary/greeter/blob/6f42e8100a1e132de5a8ceb1065f91ecb3b70092/src/CapsLockRevealer.vala#L65

**Comment by CodingSpiderFox** [comment]
Yes, I think this might be sufficient

Am Mi., 25. Sept. 2019 um 17:22 Uhr schrieb Cassidy James Blaede <
notifications@github.com>:

**Comment by cassidyjames** [comment]
@CodingSpiderFox I spoke with @danrabbit and it looks like we do use `<small>` throughout the rest of the OS. What physical size is your display, out of curiosity?

**Comment by CodingSpiderFox** [comment]
The problem appears on all displays I used so far. 22", 24", 12,5"

**Comment by danrabbit** [comment]
Now that we have caps and num indicators in the panel, I wonder if this UI should go away. Thoughts from @elementary/ux ?

**Comment by cassidyjames** [comment]
@danrabbit ah yeah we could enable those by default in the greeter session, maybe? Or do the whole if-the-user-enabled-them-in-their-session thing with AccountsService.
